### DISCUSSION TITLE
Is there a way to hide code cells converting from ipynb to docx?

### DISCUSSION THREAD

**Discussion by jfjvallaris** [quarto-dev/quarto-cli#5467] [Q&A] [answered]
Hi,

is there a way to hide code cells converting from ipynb to docx?

i am using a ipynb in vscode file where

i have:
```
---
title: Python analisys.
format:
  docx:
    code-annotations: false
---
```

but the output still showing the contain of code box

thanks

**Comment by mcanouil** [comment, marked as answer]
`code-annotations` is for annotating code, see <https://quarto.org/docs/authoring/code-annotation.html>.
It does not seem this option has anything to do with your issue.

Instead, i suggest you take a look at Jupyter option used in Quarto, such as `echo: false` (sub-key of `execute` for global effect), see <https://quarto.org/docs/reference/cells/cells-jupyter.html>.

**Reply by jfjvallaris** [reply]
Thanks a lot. I solved my problem with thah information
